README.md

```diff
@@ -8,6 +8,19 @@ For more information on building and developing the core of Scala, read on! Plea
 We're still using Jira for issue reporting, so please [report any issues](https://issues.scala-lang.org) over there.
 (We would love to start using GitHub Issues, but we're too resource-constrained to take on this migration right now.)
 
+# Get in touch!
+If you need some help with your PR at any time, please feel free to @-mention anyone from the list below, and we will do our best to help you out:
+
+ username     |      full name       | talk to me about...                               |
+--------------|----------------------|---------------------------------------------------| 
+ adriaanm     | Adriaan Moors        | anything (type checker, pattern matcher, CI)      |
+ gkossakowski | Grzegorz Kossakowski | infrastructure, incremental compilation, back-end |
+ retronym     | Jason Zaugg          | Java 8 lambdas, tricky bug detective work         |
+ Ichoran      | Rex Kerr             | the collections library                           |
+ lrytz        | Lukas Rytz           | optimizer, named & default arguments              |
+ dickwall     | Dick Wall            | process & documentation                           |
```

Comment:
In the interest or demonstrating our humanity (or unicornity), how about:


                                                                                                  | username                                                              | talk to me about...                               |
--------------------------------------------------------------------------------------------------|---------------------------------------------------------------|---------------------------------------------------|
 !<img src="https://avatars.githubusercontent.com/adriaanm" height="50px" title="Adriaan Moors"/> | [`@adriaanm`](https://github.com/adriaanm)                    | anything (type checker, pattern matcher, CI)      |
 !<img src="https://avatars.githubusercontent.com/gkossakowski" height="50px" title="Grzegorz Kossakowski"/> | [`@gkossakowski`](https://github.com/gkossakowski) | infrastructure, incremental compilation, back-end |
 !<img src="https://avatars.githubusercontent.com/retronym" height="50px" title="Jason Zaugg"/>   | [`@retronym`](https://github.com/retronym)                    | Java 8 lambdas, tricky bug detective work         |
 !<img src="https://avatars.githubusercontent.com/Ichoran" height="50px" title="Rex Kerr"/>       | [`@Ichoran`](https://github.com/Ichoran)                      | the collections library, performance              |
 !<img src="https://avatars.githubusercontent.com/lrytz" height="50px" title="Lukas Rytz"/>       | [`@lrytz`](https://github.com/lrytz)                          | optimizer, named & default arguments              |